TRENZ.Docs.API/Services/NavTreeProvider.cs

```diff
@@ -18,18 +18,23 @@ public NavTreeProvider(INavNodeFlaggingService flaggingService, INavNodeOrdering
     }
 
     /// <inheritdoc />
-    public async Task<NavTree> RebuildAsync(List<IndexFile> indexFiles, CancellationToken cancellationToken = default)
+    public async Task<NavTree> RebuildAsync(List<ISourceFile> files, CancellationToken cancellationToken = default)
     {
         var root = new Dictionary<string, NavNode>();
 
-        foreach (var file in indexFiles)
+        foreach (var file in files)
         {
-            var path = file.location.Split(NavNode.Separator);
+            var path = file.RelativePath.Split(Path.DirectorySeparatorChar);
             var currentPath = new List<string>();
             var node = root;
             for (var i = 0; i < path.Length; i++)
             {
                 var part = path[i];
+                if (part.EndsWith(".md"))
```

Comment:
do we need to special-case this? can't we simply strip off the extension?

Reply:
Then `.order` and `.authz` are considered the same node. I also wanted to preserve the file extension for pictures and other files (expect `.md`-files, which should appear without extension in the tree nodes).